### PR TITLE
Duplicate result records for results with multiple qualifier codes (WQP-1586)

### PR DESCRIPTION
**Pull request by kkehl-usgs**
Initial effort to combine result_meas_qual_codes and remove duplicates.

### CODE REVIEW

**Comment by kkehl-usgs** [comment]
The current test failures are due to a pre-existing problem with tl_2019_us_county_geopkg that is under investigation, but aren't related to this pull request.

File: src/main/resources/sql/result/resultStoretw.sql [hunk 1]
```diff
@@ -82,4 +82,50 @@ select 3 data_source_id,
...
+    and t1.site_id = t2.site_id
+    and t1.analytical_method = t2.analytical_method
+    and t1.analytical_method = t2.analytical_method
+    and t1.characteristic_type = t2.characteristic_type
+    and t1.site_type = t2.site_type
+    and t1.huc = t2.huc
+    and t1.governmental_unit_code = t2.governmental_unit_code
+    and t1.geom = t2.geom
+    and t1.result_meas_qual_code is not null
+    and t2.result_meas_qual_code is not null;
```
Comment (orosz-usgs) [inline src/main/resources/sql/result/resultStoretw.sql:131]: Looks good. The one thing I would check is to verify that this is called in an transaction and would rollback on
an error. I do not know the default for JDBC template or exactly how it is configured in the service.

Comment (kkehl-usgs) [reply src/main/resources/sql/result/resultStoretw.sql]: @orosz-usgs  -- could you take another look?
Looks like for springboot projects, transactions are enabled by default.  You just use the Transactional annotation on a public method.   I wrapped my changes in a transaction so if there is a problem there it doesn't interfere with the data getting inserted.

Comment (orosz-usgs) [reply src/main/resources/sql/result/resultStoretw.sql]: Thanks for making the update. Look good. This should make it a little more robust.

**Pull request merged**